### PR TITLE
[Editor]Refactor ObjectListItem with MUI components

### PR DESCRIPTION
**Pull request by AhmetTanakol**

### CODE REVIEW

**Comment by AhmetTanakol** [comment]
Updated

File: packages/editor/src/EditorIde.tsx [hunk 1]
```diff
@@ -1,19 +1,35 @@
...
-        schema={schema}
-        filterPredicate={filterPredicate}
-        labelProvider={labelProvider}
-        imageProvider={imageProvider}
-      />
-    </div>
+    <MuiThemeProvider theme={theme}>
+      <div>
+        <TreeRenderer
+          className={editor}
```
Comment (edgarmueller) [inline packages/editor/src/EditorIde.tsx]: Why don't we use `withStyles` here?

File: packages/editor/src/EditorIde.tsx [hunk 2]
```diff
@@ -1,19 +1,35 @@
...
-      <TreeRenderer
-        uischema={uischema}
-        schema={schema}
-        filterPredicate={filterPredicate}
-        labelProvider={labelProvider}
-        imageProvider={imageProvider}
-      />
-    </div>
+    <MuiThemeProvider theme={theme}>
+      <div>
```
Comment (edgarmueller) [inline packages/editor/src/EditorIde.tsx]: Superfluous div

File: packages/editor/src/tree/TreeRenderer.tsx [hunk 3]
```diff
@@ -95,6 +101,58 @@ const Master = (
 
 const isNotTuple = (schema: JsonSchema7) => !Array.isArray(schema.items);
 
+const styles: StyleRulesCallback<'jsfTreeMasterDetail' |
```
Comment (edgarmueller) [inline packages/editor/src/tree/TreeRenderer.tsx]: We don't need the `jsf` prefix anymore, I'd remove it

**Comment by AhmetTanakol** [comment]
Updated

**Pull request merged**